documentation/source/users/rmg/input.rst

```diff
@@ -230,8 +230,64 @@ The following is an example of a typical species item, based on methane using SM
 			"""
 	)
 
-.. _reactionsystem:
+.. _forbidden_structures:
+
+Forbidden Structures
+=====================
+
+RMG exlores a wide varitey of structures as a builds a mechanism.
+Sometimes, RMG makes nonsensical, unphysical, or unstable species that make their way into the core.
+
+Luckily, RMG has a remedy to prevent these undesirable structures from making their way into your mechanism.
+RMG's ForbiddenStructures database contains structures that are forbidden from entering the core.
+
+While this database forbids many undesirable structures, it may not contain a species
+or structure you would like to exclude from your model. If you would like to add a forbidden structure for your RMG job,
```

Comment:
To "_add_ a forbidden structure to your job" could be misinterpreted. We want to forbid a structure from our model, not add it.